components/zoho_creator/zoho_creator.app.mjs

```diff
@@ -1,11 +1,46 @@
+import { axios } from "@pipedreamhq/platform";
+
 export default {
   type: "app",
   app: "zoho_creator",
   propDefinitions: {},
   methods: {
-    // this.$auth contains connected account data
-    authKeys() {
-      console.log(Object.keys(this.$auth));
+    _authToken() {
+      return this.$auth.oauth_access_token;
+    },
+    _makeRequestConfig() {
+      const authToken = this._authToken();
+      const headers = {
+        "Authorization": `Zoho-oauthtoken ${authToken}`,
+        "User-Agent": "@PipedreamHQ/pipedream v0.1",
+      };
+      return {
+        headers,
+      };
+    },
+    _apiUrl() {
+      return "https://creator.zoho.com/api/v2";
```

Comment:
You'll need to use the value of `this.$auth.base_api_url` as the domain, since users in different Zoho regions will use a different API endpoint:

```
_apiUrl() {
  return `https://creator.${this.$auth.base_api_uri}/api/v2/applications`
}
```

I'd recommend creating accounts in multiple regions (e.g. US and EU) to confirm the source works end-to-end.

Reply:
Thank you so much. I thought i was using .eu domain for testing, but somehow missed it.